packages/syft/src/syft/core/node/vm/vm.py

```diff
@@ -92,3 +93,15 @@ def _register_frameworks(self) -> None:
         #                 "Framework already imported. Why are you importing it twice?"
         #             ))
         #         self.frameworks.attrs[name] = ast
+
+    def __hash__(self) -> int:
```

Comment:
Those needs to be removed

Reply:
Why does this need to be removed? And are you asking or saying?